layers/+window-management/exwm/packages.el

```diff
@@ -30,7 +30,39 @@
             :step pre)
       (exwm :location (recipe :fetcher github
                               :repo "ch11ng/exwm")
-            :step pre)))
+            :step pre)
```

Comment:
`:step` is a keyword we use for `bootstrap` package.
Is there any difference if you remove `:step pre` from `xelb` and `exwm`?

Reply:
There isn't a difference if either `exwm` or `xelb` is already installed, but I *think* if Spacemacs is being installed for the first time, putting it in `:step pre` ensures that that `exwm` doesn't completely fail.

Still, this is out of the scope of this PR.

Reply:
I've posted the question whether to allow `:step pre` for non-bootstrap packages to maintainers.
But since this is a minor issue, I will merge this PR now.